src/crimson/osd/pg.cc

```diff
@@ -749,6 +770,7 @@ seastar::future<Ref<MOSDOpReply>> PG::do_osd_ops(
     logger().debug("encountered the legacy error handling path!");
     return handle_failed_op(e.code(), std::move(obc), *ox_deleter, *m);
   });
+  return crimson::ct_error::eagain::make();
```

Comment:
why?

Reply:
My bad, this return statement should actually be inside the handling of `read_errorator::all_same_way` right after the call to `rep_repair_primary_object()`